test/system/800-config.bats

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Test specific configuration options and overrides
+#
+
+load helpers
+
+@test "podman CONTAINERS_CONF - CONTAINERS_CONF in conmon {
```

Comment:
Broken, missing end quote, this will fail. But please stay tuned for further review feedback.

Reply:
Weird... I'm not sure how I missed that when I ran the tests.  Anyway, fixed now.